src/neptune/metadata_containers/metadata_containers_table.py

```diff
@@ -157,11 +161,12 @@ def __init__(
         self,
         backend: NeptuneBackend,
         container_type: ContainerType,
-        entries: List[LeaderboardEntry],
+        entries: Generator[LeaderboardEntry, None, None],
     ):
         self._backend = backend
         self._entries = entries
         self._container_type = container_type
+        self._iterator = iter(entries if entries else ())
 
     def to_rows(self) -> List[TableEntry]:
         return [
```

Comment:
Could it be implemented with `list(self)` now? 😮

Reply:
Done